packages/clerk-js/src/ui/components/Subscriptions/SubscriptionsList.tsx

```diff
@@ -164,57 +159,53 @@ export function SubscriptionsList({
                     )}
                   </Text>
                 </Td>
-                <Td
-                  sx={_ => ({
-                    textAlign: 'right',
-                  })}
-                >
-                  <Button
-                    aria-label='Manage subscription'
-                    onClick={event => openSubscriptionDetails(event)}
-                    variant='bordered'
-                    colorScheme='secondary'
-                    isDisabled={!canManageBilling}
-                    sx={t => ({
-                      width: t.sizes.$6,
-                      height: t.sizes.$6,
-                    })}
-                  >
-                    <Icon
-                      icon={CogFilled}
-                      sx={t => ({
-                        width: t.sizes.$4,
-                        height: t.sizes.$4,
-                        opacity: t.opacity.$inactive,
-                      })}
-                    />
-                  </Button>
-                </Td>
               </Tr>
             ))}
           </Tbody>
         </Table>
       )}
 
-      {(commerceSettings.billing.user.hasPaidPlans && subscriberType === 'user') ||
-      (commerceSettings.billing.organization.hasPaidPlans && subscriberType === 'organization') ? (
-        <ProfileSection.ArrowButton
-          id='subscriptionsList'
-          textLocalizationKey={subscriptionItems.length > 0 ? arrowButtonText : arrowButtonEmptyText}
-          sx={[
-            t => ({
-              justifyContent: 'start',
-              height: t.sizes.$8,
-            }),
-          ]}
-          leftIcon={subscriptionItems.length > 0 ? ArrowsUpDown : Plus}
-          leftIconSx={t => ({
-            width: t.sizes.$4,
-            height: t.sizes.$4,
-          })}
-          onClick={() => void navigate('plans')}
-        />
-      ) : null}
+      <ProfileSection.ButtonGroup id='subscriptionsList'>
+        {billingPlansExist ? (
+          <ProfileSection.ArrowButton
+            id='subscriptionsList'
+            textLocalizationKey={subscriptionItems.length > 0 ? switchPlansLabel : newSubscriptionLabel}
+            sx={[
+              t => ({
+                justifyContent: 'start',
+                height: t.sizes.$8,
+                width: isManageButtonVisible ? 'unset' : undefined,
```

Comment:
whats the goal were after here? span full width if alone, justify between if there are two? Can this be handled via the parent group container vs needing to do this width changes/unsetting here?